lib/wab/impl/init.rb

```diff
@@ -25,13 +25,16 @@ def initialize(path, config)
         types      = config[:rest] || []
         config_dir = "#{path}/config"
         lib_dir    = "#{path}/lib"
+        init_site  = config[:site]
 
-        raise WAB::Error.new('At least one record type is required for new or init.') if (types.nil? || types.empty?) && !config[:site]
+        if (types.nil? || types.empty?) && !init_site
+          raise WAB::Error.new("At least one record type is required for 'new' or 'init'.")
+        end
```

Comment:
Funny, that is the way I had it but changed because I figured you would put in a PR to change it anyway.

Reply:
not when the resulting line is too long.. (more than 80-90 chars) :smiley: